db-scripts.js

```diff
@@ -0,0 +1,34 @@
+"use strict";
+
+const pg = require("pg");
+
+const AppConstants = require("./app-constants").init();
+const db = require("./db");
+
+async function setup() {
+  const client = new pg.Client({
+    database: AppConstants.DATABASE_NAME,
+    host: AppConstants.DATABASE_HOST,
+    port: AppConstants.DATABASE_PORT,
+  });
+
+  try {
+    await client.connect();
+    await client.query("DROP TABLE users;");
```

Comment:
This isn't meant to go to production, or maybe ever get committed into the repository at all, but if we do, we should make it far more sophisticated and maybe make a backup of the table here.